docs/source/markdown/podman-create.1.md

```diff
@@ -776,7 +776,7 @@ If container is running in --read-only mode, then mount a read-write tmpfs on /r
 
 #### **--replace**=**true**|**false**
 
-If another container with the same name already exists, replace and remove it. The default is **false**.
+If another container with the same name already exist, replace and remove it. The default is **false**.
```

Comment:
```suggestion
If another container with the same name already exists, replace and remove it. The default is **false**.
```

Reply:
Sorry, I screwed this one up earlier, "exists" is correct for third person singular.  "exist" is good for you, we, they, or you.